src/api/app/models/event_subscription/find_for_event.rb

```diff
@@ -52,6 +52,9 @@ def subscriptions(channel = :instant_email)
               channel: default_subscription.channel,
               subscriber: receiver
             )
+          elsif channel == :web && receiver.instance_of?(Group) && receiver.web_users.any? { |u| EventSubscription.for_subscriber(u).find_by(options).present? }
```

Comment:
How can this be a group if we expand them above?

Reply:
we dont expand them when its for the web channel

Reply:
but maybe we should instead of the step we doing here? hmmm